terraform/cloud-functions/per-project/test/per_project_e2e_test.go

```diff
@@ -160,8 +160,13 @@ func TestPerProjectEndToEndDeployment(t *testing.T) {
 	})
 
 	defer test_structure.RunTestStage(t, "teardown", func() {
+		logger.Log(t, "TEST STAGE: TEARDOWN")
+		logger.Log(t, "----------------------------------------------------------")
 		terraformOptions := test_structure.LoadTerraformOptions(t, terraformDir)
+		terraformOptions.Logger = logger.Discard
```

Comment:
I don't think we should discard the teardown logs -- these are useful for debug as we need to ensure teardown is clean and understand why if not.

Reply:
Done